src/main/java/ladder/model/NameList.java

```diff
@@ -0,0 +1,50 @@
+package ladder.model;
+
+import ladder.model.utils.StringToListConverter;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class NameList {
+    private static final String NAME_PATTERN = "^[a-zA-Z]*$";
+    private static List<String> nameList;
+
+    public NameList(String nameList) {
+        this.nameList = StringToListConverter.toList(nameList);
+        checkDuplicate();
+        checkPattern();
+        checkLength();
+    }
+
+    private static void checkDuplicate() {
+        if (nameList.size() != nameList.stream().distinct().count()) {
+            throw new IllegalArgumentException("이름은 중복일수 없습니다.");
+        }
+    }
+
+    private static void checkPattern() {
+        nameList.stream()
+                .filter(name -> !Pattern.matches(NAME_PATTERN, name))
+                .findFirst()
+                .ifPresent(name -> {
+                    throw new IllegalArgumentException("아이디는 영어 대소문자 / 숫자만 포함 가능.");
```

Comment:
용어도 통일성 있게 '이름'이라고 하면 어떨까요
```suggestion
                    throw new IllegalArgumentException("이름은 영어 대소문자 / 숫자만 포함 가능.");
```